src/test/scala/is/hail/vds/JoinSuite.scala

```diff
@@ -21,7 +21,7 @@ class JoinSuite extends SparkSuite {
     val right = hc.importVCF("src/test/resources/joinright.vcf")
 
     // make sure joined VDS writes
-    left.join(right).write(joinedPath)
+    left.union_cols(right).write(joinedPath)
```

Comment:
Should this method be unionCols?

Reply:
yep

Reply:
This is not fixed yet.

Reply:
ack, should have refactored instead of renaming manually